backend/src/routes/preferences.js

```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+var mongoose = require('mongoose');
+const Preferences = require("../models/preferencesModel");
+const router = express.Router();
+
+/**
+ * Get Preferences Object from Preferences Collection by Id of respective user
+ * 
+ * Route: GET /api/user/:userId/preferences where userId is the ID of user
+ */
+router.get('/:userId/preferences', async (req, res) => {
+    try {
+        const preferences = await Preferences.findOne({userID: req.params.userId});
+        if(preferences){
+            res.status(200).json(preferences);
+        }else{
+            res.status(404).json({"status": 404, "message": "Did not match any user!"});
```

Comment:
We should have consistent response bodies :joy: . If you have a preference for this format, we can do that.

Reply:
Agreed T_T. Hm I think it looks neat with the `async-await` and also handles internal server errors as well in case there is some validation mistake or such. But also I've not used JS in a long while so not sure whats meant to be best practices.

Reply:
I'll format to yours!